public/talos/v1.11/platform-specific-installations/cloud-platforms/scaleway.mdx

```diff
@@ -23,11 +21,12 @@ The process to run a Talos node in Scaleway is as follows:
 
   <CodeBlock lang="sh">
     {`
-   wget "https://factory.talos.dev/image/376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba/${release_v1_11}/scaleway-amd64.raw.zst"
+   wget "https://factory.talos.dev/image/18d0321d7fb289f707a76e1deeaa5c97e62209722cbf4bc533a5d51eb666885f/v{ release_v1_11 }/scaleway-amd64.raw.zst"
```

Comment:
```suggestion
   wget "https://factory.talos.dev/image/18d0321d7fb289f707a76e1deeaa5c97e62209722cbf4bc533a5d51eb666885f/v{ release_v1_11 }/scaleway-amd64.qcow2"
```

You can generate the qcow2 image directly from the factory and skip the whole `Decompress and convert the image` section. You might even be able to get scaleway to import the image directly without need for a bucket.